index.js

```diff
@@ -63,4 +63,5 @@ E.prototype = {
   }
 };
 
-module.exports = E;
+module.exports.default = E;
+module.exports.emitter = new E();
```

Comment:
Not really. 

When a file is required it only parses the file once and each require() after that is just a reference to the first time it is parsed. Even if you require the file 1000 times, the object is only instantiated once.

I've done a bit of testing on this now though and I can't figure out how to make it work the way that it is supposed to without using es6 export syntax.

I think we should just go with the extra file method for the sake of guaranteed backwards compatibility.